packages/tradershub/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx

```diff
@@ -18,10 +18,12 @@ const LeadingIcon = () => (
 
 const AddedCTraderAccountsList = () => {
     const { data: cTraderAccounts } = useCtraderAccountsList();
-    const { data: activeTrading } = useActiveTradingAccount();
+    const { data: activeTradingAccount } = useActiveTradingAccount();
     const { show } = Provider.useModal();
-    const account = cTraderAccounts?.find(account => account.is_virtual === activeTrading?.is_virtual);
+    const account = cTraderAccounts?.find(account => account.is_virtual === activeTradingAccount?.is_virtual);
     const isVirtual = account?.is_virtual;
+    const demoTitle = activeTradingAccount?.is_virtual ? 'Demo' : '';
```

Comment:
I think it's better to change the title in the PlatformDetails instead of manipulating it here in the component level

```
export const PlatformDetails = {
    ctrader: {
        icon: (width?: number) => <PlatformIcon icon='CTrader' width={width} />,
        link: 'https://onelink.to/hyqpv7',
        platform: 'ctrader' as TPlatforms.OtherAccounts,
        title_real: 'Deriv cTrader',
        title_demo: 'Deriv cTrader demo',
    },
    ...
```

or passing the argument to the platfromDetails to change the title:

export const PlatformDetails = (isDemo: boolean = false) => {
    ctrader: {
        icon: (width?: number) => <PlatformIcon icon='CTrader' width={width} />,
        link: 'https://onelink.to/hyqpv7',
        platform: 'ctrader' as TPlatforms.OtherAccounts,
        title: isDemo?  'Deriv cTrader Demo' :  'Deriv cTrader',
    },

Reply:
@mahdiyeh-deriv  The first approach still we need to repeat the logic in the component level , the second approach we can't change the title in that level cause we are using the title in other places which demo is not needed